keylime/shared_data.py

```
@@ -356,12 +368,10 @@ def cleanup(self) -> None:
             )
             return
 
-        logger.debug("Shutting down SharedDataManager")
         try:
             self._manager.shutdown()
-            logger.info("SharedDataManager shutdown complete")
         except Exception:
-            logger.exception("Error during SharedDataManager shutdown")
+            pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)